moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -109,6 +110,15 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
     return setJointValueTarget(js_msg);
   }
 
+  bool setStateValueTarget(const std::string& state_str)
```

Comment:
@willcbaker I didn't realize that you just recently added this function.
This is highly dangerous, as it allows to setting of joints that are not part of the JointModelGroup!

Reply:
Exposing a publicly available c++ move group method
http://docs.ros.org/jade/api/moveit_ros_planning_interface/html/classmoveit_1_1planning__interface_1_1MoveGroup.html#ad35630dd853a734de3580390facf2c9b
iirc it was (is?) the only way to set a desired multi dof joint state.